doc/source/whatsnew/v0.17.0.txt

```diff
@@ -46,12 +46,58 @@ Backwards incompatible API changes
 
 .. _whatsnew_0170.api_breaking.other:
 
+Changes to convert_objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+- ``DataFrame.convert_objects`` keyword arguments have been shortened. (:issue:`10265`)
+
+=====================   =============
+Old                     New
+=====================   =============
+``convert_dates``       ``datetime``
+``convert_numeric``     ``numeric``
+``convert_timedelta``   ``timedelta``
+=====================   =============
+
+- Coercing types with ``DataFrame.convert_objects`` is now implemented using the
+keyword argument ``coerce=True``.  Previously types were coerced by setting a
+keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
+
+  Previously data was coerced to a common dtype before serialisation, which for
+  example resulted in integers being serialised to floats:
+
+  .. code-block:: python
+
+    In [1]: df = pd.DataFrame({'i': ['1','2'], 'f': ['apple', '4.2']})
+    In [2]: df.convert_objects(convert_numeric='coerce')
+
+  Now the ``coerce`` keyword must be explicitly used.
+
+  .. code-block:: python
```

Comment:
Presumably the old code which doesn't work any more should just be in a standard python block?

```
  .. code-block:: python
  
    In [2]: df.convert_objects(convert_numeric='coerce')
```